payjoin-directory/src/db.rs

```diff
@@ -99,6 +117,6 @@ impl DbPool {
     }
 }
 
-fn channel_name(pubkey_id: &str, channel_type: &str) -> String {
-    format!("{}:{}", pubkey_id, channel_type)
+fn channel_name(pubkey_id: &ShortId, channel_type: &str) -> Vec<u8> {
```

Comment:
This helper is no longer that helpful IMO and could be replaced by direct calls to `column_key()` in later commits.